tests/utilities/imageAlreadyInDbCheck.spec.ts

```diff
@@ -190,7 +191,8 @@ describe("utilities -> imageAlreadyInDbCheck", () => {
 
     try {
       await imageAlreadyInDbCheck(null, testNewImagePath);
-    } catch (error: any) {
+    } catch (error: unknown) {
+      if (!(error instanceof ApplicationError)) return;
```

Comment:
Make the test fail in such a case.

Reply:
> Make the test fail in such a case.

if (!(error instanceof Error)) throw new Error;

Sorry for asking but will it work?

Reply:
This is not idiomatic. 
Use [Vitest error assertions](https://vitest.dev/api/expect.html#tothrowerror) instead.